cpp/src/arrow/util/compression_zstd.cc

```diff
@@ -180,12 +189,27 @@ Status ZSTDCompressor::End(int64_t output_len, uint8_t* output, int64_t* bytes_w
   *should_retry = ret > 0;
   return Status::OK();
 }
+} // namespace
 
 // ----------------------------------------------------------------------
 // ZSTD codec implementation
 
+ZSTDCodec::ZSTDCodec() : Codec(), zstdCompressionLevel_(kZSTDDefaultCompressionLevel) {
+  const char* envZstdCompressionLevel = getenv("ARROW_ZSTD_COMPRESSION_LEVEL");
+  if (envZstdCompressionLevel != NULLPTR) {
```

Comment:
nullptr is fine in .cc files.

Reply:
I didn't know this, does that mean that `nullptr` shouldn't be used in .h files?